src/app/globals.css

```
@@ -1,26 +1,1108 @@
 @import "tailwindcss";
+@plugin "@tailwindcss/typography";
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)